src/App.js

```diff
@@ -58,19 +70,18 @@ class App extends Component {
           <h2 className="page-name">Quizzelydoo</h2>
         </div>
         {isQuizIntro ? (
-          <div className="Introduction">
-            <Adbox />
+          <Transition in={this.state.in} timeout={400} className="exiting">
```

Comment:
Excellent use of the Transition module. You're very close. The thing you missed is that you don't need an "in" state. The "in" in <Transition in={}> should be filled with a value that tells transition when that item is visible. In this case the item is only visible when "isQuizIntro" is true. So just fill it with that and get rid of all that other "in" stuff.